### ISSUE TITLE
[EPIC] OpenPulse Documentation

### ISSUE THREAD

**Issue by lcapelluto** [Qiskit/qiskit-tutorials#869]
Each item in the checklist needs its own issue, and that issue needs to be linked here before starting.

#### Pulse Basics

 - [x] getting started
 - [x] creating instructions
    - [x] What are channels?
    - [x] building pulses
        - [x] using the `pulse_lib` or using `numpy`
        - [x] using parametric pulses (this method should be encouraged)
        - [x] How to update the modulation frequency of a channel (or SetFrequency, later)
 - [x] how to build a schedule (explain the syntax, explain multiple ways to build the schedule, explain schedule operations and composition)
 - [x] how to add measurements, including explaining `meas_map` with a link to page about `meas_level`s (using a custom discriminator?)
 - [x] getting data about the system
    - [x] configuration
    - [x] defaults and InstructionScheduleMap
    - [x] properties
 - [x] Using the scheduler (default scheduling method, demo how to use it, explain key features such as acquisition alignment) - IP
    - [x] different methods
    - [x] custom scheduling with InstMap
 - [ ] how to use execute (special args: frequencies passed that way, meas levels, etc)
 - [ ] troubleshooting: this is stuff that people ask about over time. Hopefully, everything will have been covered in the above, but there should still be a landing spot for this

#### Pulse Advanced

 - [ ] special results types
    - [ ] different meas levels
    - [ ] extracting data from the results objects for each meas level
 - [ ] help with typical use cases
    - [ ] how to vary parameters
 - [ ] Extending the scheduler
 - [ ] more advanced schedule building
    - [ ] what are control channels? how to get data about them and how to use them?
    - [ ] how to build and understand the system hamiltonian (hopefully, can mostly link to other resources)
    - [ ] What are frames? What is a frame change? How to add one? Why might you add a FC on derived channels? This ties into #3467 also

### Aer
 - [x] Using the pulse simulator

## Tutorials for OpenPulse should demonstrate:
 - frequency sweep
 - basic implementation of each Pauli gate using pulse
 - basic CX bringup tuning CR pulses
 - measuring t1
 - measuring t2
 - dynamical decoupling
 - qubit spec (chevron)
 - creating a custom discriminator

Most of these are done in various places, and may need cleaning up, updating to use the latest code, or more thorough explanations.

**Comment by lcapelluto** [comment]
Please assign me:)

**Comment by nonhermitian** [comment]
I think a lot of pulse stuff needs to be reworked as the tutorials within the docs cannot access the IQX.

**Comment by Eric-Arellano** [comment]
Closing in preparation for archiving this repo in favor of the tutorials living directly in Qiskit (aka Terra), qiskit-algorithms, and qiskit-aer.

I believe this is out of date too.
